Tests/CommandsTests/RunCommandTests.swift

```diff
@@ -40,6 +40,25 @@ final class RunCommandTests: CommandsTestCase {
         XCTAssert(stdout.contains("Swift Package Manager"), "got stdout:\n" + stdout)
     }
 
+// echo.sh script from the toolset won't work on Windows
+#if !os(Windows)
```

Comment:
suggestion: can we create an equivalent target that will run on Windows to ensure the functions work on that host platform?

Reply:
I'm not sure, I don't have experience with Windows and don't have means to test on Windows. IIRC the test suite doesn't even run on the Windows CI job. I'd prefer to do this in a separate PR.